src/app/ui/app/App.tsx

```diff
@@ -1,31 +1,48 @@
-import { getCookies } from '@/app/lib/getCookies';
-import { FeatureCards } from '@/components/FeatureCards/FeatureCards';
-import { Snackbar } from '@/components/Snackbar/Snackbar';
+'use client';
+
 import { WelcomeScreen } from '@/components/WelcomeScreen/WelcomeScreen';
-import { Widgets } from '@/components/Widgets/Widgets';
 import type { StarterVariantType } from '@/types/internal';
-import { PixelBg } from '@/components/illustrations/PixelBg';
+import { WidgetContainer } from '@/components/Widgets';
+import { TrackingAction, TrackingCategory } from '@/const/trackingKeys';
+import { EventTrackingTool } from '@/types/userTracking';
+import { useWelcomeScreen } from '@/hooks/useWelcomeScreen';
+import { useUserTracking } from '@/hooks/userTracking';
 
 export interface AppProps {
   starterVariant: StarterVariantType;
+  children: React.ReactNode;
+  isWelcomeScreenClosed: boolean;
 }
 
-const App = ({ starterVariant }: AppProps) => {
-  const { activeTheme, welcomeScreenClosed } = getCookies();
-  const isWelcomeScreenClosed = welcomeScreenClosed === 'true';
+const App = ({ starterVariant, isWelcomeScreenClosed, children }: AppProps) => {
+  const { trackEvent } = useUserTracking();
+
+  const welcomeScreen = useWelcomeScreen(isWelcomeScreenClosed);
+
+  const handleWelcomeScreenEnter = () => {
```

Comment:
We have lost the hover effect on the WelcomeScreen, was it expected?

Reply:
done